libraries/app/api.cpp

```diff
@@ -302,26 +302,31 @@ namespace graphene { namespace app {
        const auto& db = *_app.chain_database();
        FC_ASSERT( limit <= 100 );
        vector<operation_history_object> result;
-       const auto& stats = account(db).statistics(db);
-       if( stats.most_recent_op == account_transaction_history_id_type() ) return result;
-       const account_transaction_history_object* node = &stats.most_recent_op(db);
+       account_statistics_object stats;
+       try { stats = account(db).statistics(db); } catch(...) { return result; }
+       const account_transaction_history_object* node;
+       try { node = &stats.most_recent_op(db); } catch(...) { return result; }
        if( start == operation_history_id_type() )
           start = node->operation_id;
+       if( start.instance.value > node->operation_id.instance.value )
```

Comment:
Please merge this if with the previous one.

Reply:
please explain more. thx.

Reply:
```if (a) do_x(); if (b) do_x();```

is equivalent to

```if (a || b) do_x();```

(in this case with the specific condition that do_x() implies !b).

Reply:
git it and changed. thanks.